boogiestats/boogie_ui/views.py

```diff
@@ -505,6 +508,34 @@ def get_redirect_url(self, *args, **kwargs):
         return reverse("player", kwargs={"player_id": player_id})
 
 
+@login_required(login_url="/login/")
```

Comment:
Good catch! This is not a normal scenario -- the endpoint should only work with POST method because it's modifying state. I will add a guard against it on `add/remove_rival` and a redirect in that case. 

BTW, the `@login_required` without a `login_url` used a fallback to a non-existent default route.

Reply:
I added guards for request method and rolled out current version to the staging server

Reply:
replaced manual guard with django helper decorator `@require_POST`